src/EventListener/MergeHttpHeadersListener.php

```diff
@@ -92,8 +92,10 @@ private function mergeHttpHeaders(Response $response)
                 header_remove($name);
             }
 
-            // Do not replace existing headers as the response object has a higher priority
-            $response->headers->set($name, trim($content), false);
```

Comment:
If you provide ```false``` as 3rd argument it will indeed prevent from replacing the header but it will add it to the existing collection instead.